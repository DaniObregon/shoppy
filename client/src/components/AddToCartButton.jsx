import React from "react";
import { chakra } from "@chakra-ui/react";
import { useSelector } from "react-redux";

export const AddToCartButton = ({ id }) => {
  // Obtener el role_id desde el estado de Redux
  const role_id = useSelector((state) => state.userInfo.role_id);

  // Evita la propagación del clic hacia el contenedor de la tarjeta
  const handleAddToCart = (event) => {
    event.stopPropagation();

    // Lógica para agregar el producto al carrito
    if (role_id === 2) {
      // Usuario con rol 'admin'
      console.log(`Producto con id ${id} agregado al carrito.`);
    } else {
      // Si no es un admin
      console.error(
        "Acceso denegado: Solo los admin pueden agregar productos al carrito."
      );
    }
  };

  return (
    <chakra.button
      bg="gray.800"
      fontSize="xs"
      fontWeight="bold"
      color="white"
      px={2}
      py={1}
      rounded="lg"
      textTransform="uppercase"
      _hover={{
        bg: "gray.700",
        _dark: { bg: "gray.600" },
      }}
      _focus={{
        bg: "gray.700",
        _dark: { bg: "gray.600" },
        outline: "none",
      }}
      onClick={handleAddToCart} // Evento de clic del botón para agregar al carrito
    >
      Add to cart
    </chakra.button>
  );
};
