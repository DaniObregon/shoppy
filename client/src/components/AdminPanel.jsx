import React, { useEffect, useState } from "react";
import {
  Button,
  Box,
  Text,
  Table,
  Thead,
  Tbody,
  Tr,
  Th,
  Td,
  useDisclosure,
  Modal,
  ModalOverlay,
  ModalContent,
  ModalHeader,
  ModalCloseButton,
  ModalBody,
  FormControl,
  FormLabel,
  Input,
  ModalFooter,
  useToast,
} from "@chakra-ui/react";
import { ConfirmDeleteDialog } from "./ConfirmDeleteDialog"; // Componente de confirmación de eliminación
import { ProductTable } from "./ProductTable";
import { useNavigate } from "react-router-dom";
import { useDispatch, useSelector } from "react-redux";
import { fetchProductById, clearProduct } from "../redux/slices/productSlice";
import api from "../config/axiosConfig";

export const AdminPanel = () => {
  const navigate = useNavigate();
  const dispatch = useDispatch();
  const { data: product } = useSelector((state) => state.product);
  const [products, setProducts] = useState([]);
  const [formData, setFormData] = useState({
    id: "",
    brand: "",
    model: "",
    description: "",
    price: "",
    imgUrl: "",
    stock: "",
  });
  const [shouldReload, setShouldReload] = useState(false);
  const toast = useToast();
  const { isOpen, onOpen, onClose } = useDisclosure();

  // Estado para el diálogo de confirmación
  const {
    isOpen: isDeleteDialogOpen,
    onOpen: openDeleteDialog,
    onClose: closeDeleteDialog,
  } = useDisclosure();
  const [productToDelete, setProductToDelete] = useState(null);
  const [isDeleting, setIsDeleting] = useState(false);

  useEffect(() => {
    const fetchProducts = async () => {
      try {
        const response = await api.get("/admin-api/products");
        setProducts(response.data);
      } catch (error) {
        console.error("Error fetching products:", error);
      }
    };

    fetchProducts();
  }, []);

  useEffect(() => {
    if (shouldReload) {
      const fetchProducts = async () => {
        try {
          const response = await api.get("/admin-api/products");
          setProducts(response.data);
        } catch (error) {
          console.error("Error fetching products:", error);
        }
      };
      fetchProducts();
      setShouldReload(false);
    }
  }, [shouldReload]);

  const handleDeleteProduct = async (id) => {
    // Agregar estado de carga durante la eliminación
    setIsDeleting(true);
    try {
      await api.delete(`/admin-api/products/${id}`);
      setProducts(products.filter((product) => product.id !== id));
      toast({
        title: "Producto eliminado",
        status: "success",
        duration: 3000,
      });
      setShouldReload(true);
    } catch (error) {
      console.error("Error deleting product:", error);
      toast({
        title: "Error al eliminar",
        description: "Hubo un error al intentar eliminar el producto.",
        status: "error",
        duration: 3000,
      });
    } finally {
      setIsDeleting(false);
      closeDeleteDialog();
    }
  };

  const handleAddOrUpdateProduct = async () => {
    if (
      !formData.brand ||
      !formData.model ||
      !formData.price ||
      !formData.imgUrl ||
      !formData.name
    ) {
      toast({
        title: "Error",
        description: "Todos los campos son obligatorios.",
        status: "error",
        duration: 3000,
      });
      return;
    }

    const productData = {
      ...formData,
      stock: formData.stock || 0,
    };

    try {
      let response;
      if (formData.id) {
        response = await api.put(
          `/admin-api/products/${formData.id}`,
          productData
        );
      } else {
        response = await api.post("/admin-api/products", productData);
      }
      toast({
        title: "Producto procesado",
        status: "success",
        duration: 3000,
      });
      setFormData({
        id: "",
        brand: "",
        model: "",
        description: "",
        price: "",
        imgUrl: "",
        name: "",
        stock: "",
      });
      onClose();
      setShouldReload(true);
    } catch (error) {
      console.error("Error adding/updating product:", error);
      toast({
        title: "Error",
        description: "Hubo un error al agregar o actualizar el producto.",
        status: "error",
        duration: 3000,
      });
    }
  };

  return (
    <Box p={4}>
      <Text fontSize="2xl" fontWeight="bold" my={4}>
        Panel de Administración
      </Text>

      <Button colorScheme="teal" onClick={onOpen} mb={4}>
        Agregar Producto
      </Button>

      <ProductTable
        products={products}
        onEdit={(product) => {
          setFormData(product);
          onOpen();
        }}
        onDelete={(productId) => {
          setProductToDelete(productId);
          openDeleteDialog();
        }}
      />
      {/* Modal para agregar o modificar producto */}
      <Modal isOpen={isOpen} onClose={onClose}>
        <ModalOverlay />
        <ModalContent>
          <ModalHeader>
            {formData.id ? "Modificar Producto" : "Agregar Producto"}
          </ModalHeader>
          <ModalCloseButton />
          <ModalBody>
            <FormControl id="name" mb={4}>
              <FormLabel>Nombre</FormLabel>
              <Input
                type="text"
                value={formData.name}
                onChange={(e) =>
                  setFormData({ ...formData, name: e.target.value })
                }
                placeholder="Nombre del producto"
              />
            </FormControl>
            <FormControl id="stock" mb={4}>
              <FormLabel>Stock</FormLabel>
              <Input
                type="number"
                value={formData.stock}
                onChange={(e) =>
                  setFormData({ ...formData, stock: e.target.value })
                }
                placeholder="Stock del producto"
              />
            </FormControl>
            <FormControl id="brand" mb={4}>
              <FormLabel>Marca</FormLabel>
              <Input
                type="text"
                value={formData.brand}
                onChange={(e) =>
                  setFormData({ ...formData, brand: e.target.value })
                }
                placeholder="Marca del producto"
              />
            </FormControl>
            <FormControl id="model" mb={4}>
              <FormLabel>Modelo</FormLabel>
              <Input
                type="text"
                value={formData.model}
                onChange={(e) =>
                  setFormData({ ...formData, model: e.target.value })
                }
                placeholder="Modelo del producto"
              />
            </FormControl>
            <FormControl id="description" mb={4}>
              <FormLabel>Descripción</FormLabel>
              <Input
                type="text"
                value={formData.description}
                onChange={(e) =>
                  setFormData({ ...formData, description: e.target.value })
                }
                placeholder="Descripción del producto"
              />
            </FormControl>
            <FormControl id="price" mb={4}>
              <FormLabel>Precio</FormLabel>
              <Input
                type="number"
                value={formData.price}
                onChange={(e) =>
                  setFormData({ ...formData, price: e.target.value })
                }
                placeholder="Precio del producto"
              />
            </FormControl>
            <FormControl id="imgUrl" mb={4}>
              <FormLabel>URL de Imagen</FormLabel>
              <Input
                type="text"
                value={formData.imgUrl}
                onChange={(e) =>
                  setFormData({ ...formData, imgUrl: e.target.value })
                }
                placeholder="URL de la imagen del producto"
              />
            </FormControl>
          </ModalBody>
          <ModalFooter>
            <Button
              colorScheme="teal"
              mr={3}
              onClick={handleAddOrUpdateProduct}
            >
              {formData.id ? "Guardar Cambios" : "Agregar Producto"}
            </Button>
            <Button onClick={onClose}>Cancelar</Button>
          </ModalFooter>
        </ModalContent>
      </Modal>

      {/* Diálogo de confirmación para eliminar producto */}
      <ConfirmDeleteDialog
        isOpen={isDeleteDialogOpen}
        onClose={closeDeleteDialog}
        onConfirm={() => handleDeleteProduct(productToDelete)}
        isDeleting={isDeleting}
      />
    </Box>
  );
};
