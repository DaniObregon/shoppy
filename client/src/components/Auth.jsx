import axios from "axios";
import React from "react";
import { signInWithPopup, GoogleAuthProvider, signOut } from "firebase/auth";
import { SignUpButton } from "./SignUpButton";
import { SignOutButton } from "./SignOutButton";
import { useNavigate } from "react-router-dom";
import { auth } from "../config/firebaseConfig";
import { useDispatch, useSelector } from "react-redux";
import { login, logout } from "../redux/slices/userSlice";
import { Flex, Text, Stack } from "@chakra-ui/react";

export const Auth = ({ onAuthSuccess }) => {
  const provider = new GoogleAuthProvider();
  const navigate = useNavigate();
  const dispatch = useDispatch();

  // Obtener el estado del usuario desde Redux (obtiene los datos)
  const userInfo = useSelector((state) => state.userInfo);

  const signUpWithGoogle = async () => {
    try {
      const result = await signInWithPopup(auth, provider);
      const user = result.user;
      const token = await user.getIdToken();
      const userData = {
        name: user.displayName,
        email: user.email,
        role_id: 1, // Rol por defecto (client)
      };

      await axios.post("/api/auth/signup", JSON.stringify(userData), {
        headers: {
          "Content-Type": "application/json",
          Authorization: `Bearer ${token}`, // El token se envia en un header
        },
      });

      // Actualizar el estado global de Redux con los datos del usuario
      dispatch(
        login({
          name: user.displayName,
          email: user.email,
        })
      );

      // Si la autenticación y la creación del usuario son exitosas
      if (onAuthSuccess) {
        onAuthSuccess(user); // Llamar a la función si está definida
      }
      navigate("/"); // Redirige al home o a la ruta que prefieras después de autenticarse
    } catch (error) {
      console.error("Error durante el inicio de sesión:", error);
    }
  };

  const signOutUser = async () => {
    try {
      await signOut(auth);
      dispatch(logout()); // Despachar la acción de logout en Redux
      navigate("/");
    } catch (error) {
      console.error("Error al cerrar sesión:", error);
    }
  };

  return (
    <Flex align="center" justify="center" p={4}>
      {userInfo.name ? (
        <Stack direction="row" align="center" spacing={4}>
          <Text
            p={2}
            color="gray.600"
            whiteSpace="nowrap"
            maxWidth="200px"
            overflow="hidden"
            textOverflow="ellipsis"
          >
            Hola, {userInfo.name.split(" ")[0]}
          </Text>
          <SignOutButton onClick={signOutUser} />
        </Stack>
      ) : (
        <SignUpButton onClick={signUpWithGoogle} />
      )}
    </Flex>
  );
};
