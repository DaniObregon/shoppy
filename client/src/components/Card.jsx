import React from "react";
import { chakra, Box, Flex } from "@chakra-ui/react";

export const Card = ({ brand, model, description, price, imgUrl }) => {
  return (
    <Flex
      direction="column"
      justifyContent="center"
      alignItems="center"
      w="full"
      mx="auto"
      mb={4}
    >
      <Box
        bg="gray.300"
        h={{ base: 56, md: 64 }} //Imagen más grande para mantener la proporción
        w="full"
        rounded="lg"
        shadow="md"
        bgSize="cover"
        bgPos="center"
        style={{
          backgroundImage: `url(${imgUrl})`,
        }}
      ></Box>

      <Box
        w={{ base: "80%", md: "70%" }} // Caja de datos pequeña y centrada
        bg="white"
        _dark={{ bg: "gray.800" }}
        mt={-10} // Superpone la caja de datos en la imagen
        shadow="lg"
        rounded="lg"
        overflow="hidden"
      >
        <chakra.h3
          py={2}
          textAlign="center"
          fontWeight="bold"
          textTransform="uppercase"
          color="gray.800"
          _dark={{ color: "white" }}
          letterSpacing={1}
        >
          {brand}
        </chakra.h3>
        <chakra.p
          py={1}
          textAlign="center"
          fontWeight="medium"
          color="gray.600"
          _dark={{ color: "gray.300" }}
          fontSize="sm"
        >
          {model}
        </chakra.p>
        <chakra.p
          py={1}
          textAlign="center"
          fontWeight="medium"
          color="gray.600"
          _dark={{ color: "gray.300" }}
          fontSize="sm"
        >
          {description}
        </chakra.p>
        <Flex
          alignItems="center"
          justifyContent="space-between"
          py={2}
          px={3}
          bg="gray.200"
          _dark={{ bg: "gray.700" }}
        >
          <chakra.span
            fontWeight="bold"
            color="gray.800"
            _dark={{ color: "gray.200" }}
          >
            ${price}
          </chakra.span>
          <chakra.button
            bg="gray.800"
            fontSize="xs"
            fontWeight="bold"
            color="white"
            px={2}
            py={1}
            rounded="lg"
            textTransform="uppercase"
            _hover={{
              bg: "gray.700",
              _dark: { bg: "gray.600" },
            }}
            _focus={{
              bg: "gray.700",
              _dark: { bg: "gray.600" },
              outline: "none",
            }}
          >
            Add to cart
          </chakra.button>
        </Flex>
      </Box>
    </Flex>
  );
};
