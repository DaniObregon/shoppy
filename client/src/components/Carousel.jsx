import React, { useEffect, useState } from "react";
import { Box, Flex, Image, Text } from "@chakra-ui/react";

export const Carousel = () => {
  const slides = [
    {
      img: "https://images.pexels.com/photos/6830805/pexels-photo-6830805.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1",
    },
    {
      img: "https://images.pexels.com/photos/9863552/pexels-photo-9863552.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1",
    },
    {
      img: "https://images.pexels.com/photos/3065209/pexels-photo-3065209.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1",
    },
    {
      img: "https://images.pexels.com/photos/7990099/pexels-photo-7990099.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1",
    },
  ];
  const [currentSlide, setCurrentSlide] = useState(0);

  const slidesCount = slides.length;

  const carouselStyle = {
    transition: "all 1.1s",
    ml: `-${currentSlide * 100}%`,
  };

  const SLIDES_INTERVAL_TIME = 3000;
  const ANIMATION_DIRECTION = "right";

  useEffect(() => {
    const prevSlide = () => {
      setCurrentSlide((s) => (s === 0 ? slidesCount - 1 : s - 1));
    };

    const nextSlide = () => {
      setCurrentSlide((s) => (s === slidesCount - 1 ? 0 : s + 1));
    };

    const automatedSlide = setInterval(() => {
      ANIMATION_DIRECTION.toLowerCase() === "left" ? prevSlide() : nextSlide();
    }, SLIDES_INTERVAL_TIME);
    return () => clearInterval(automatedSlide);
  }, [slidesCount]);

  return (
    <Flex
      w="full"
      bg="#edf3f8"
      _dark={{ bg: "#3e3e3e" }}
      p={10}
      alignItems="center"
      justifyContent="center"
    >
      <Flex w="full" overflow="hidden">
        <Flex pos="relative" h="500px" w="full" {...carouselStyle}>
          {slides.map((slide, sid) => (
            <Box key={`slide-${sid}`} flex="none" boxSize="full" shadow="md">
              <Text
                color="white"
                fontSize="xs"
                p="8px 12px"
                pos="absolute"
                top="0"
                whiteSpace="nowrap"
              >
                {sid + 1} / {slidesCount}
              </Text>
              <Image
                src={slide.img}
                alt="carousel image"
                boxSize="full"
                objectFit="cover" // Ajusta la imagen para cubrir el contenedor manteniendo su relaciÃ³n de aspecto
              />
            </Box>
          ))}
        </Flex>
      </Flex>
    </Flex>
  );
};

/** IMAGENES DE ZAPATILLAS
    {
      img: "https://hips.hearstapps.com/hmg-prod/images/run-reebok-running-shoes-64e3beadbe692.jpg",
    },
    {
      img: "https://scontent.faep9-1.fna.fbcdn.net/v/t39.30808-6/221863802_537077584388553_1054731658053590263_n.jpg?_nc_cat=100&ccb=1-7&_nc_sid=127cfc&_nc_eui2=AeEEh83xoHX7uA3M3jybX6IcPs2PM-LdG08-zY8z4t0bTz2WYdy-J1oOS1LypLBSULMMu1nfH3SXI_GrnhXAvLCO&_nc_ohc=ISWDfIVXRyQQ7kNvgGrT7st&_nc_ht=scontent.faep9-1.fna&oh=00_AYC_Go5c-pXVp8JD9ebJjKMzTQxAX-PF20lZ-vedSqFgcQ&oe=66E4BDD4",
    },
    {
      img: "https://brand.assets.adidas.com/capi/enUS/Images/1839569-seo-how-to-break-in-running-shoes-body-image-2_221-681798.jpg",
    },
    {
      img: "https://portcrossfit.com/wp-content/uploads/2023/08/run-nike-running-shoes-646cdd1a19c41-1200x600.jpg",
    },
 */