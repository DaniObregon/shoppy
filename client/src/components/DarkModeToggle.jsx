import React from "react";
import { IconButton, useColorMode } from "@chakra-ui/react";
import { MoonIcon, SunIcon } from "@chakra-ui/icons";

export const DarkModeToggle = () => {
  const { colorMode, toggleColorMode } = useColorMode();

  return (
    <IconButton
      aria-label="Toggle Dark Mode"
      icon={colorMode === "light" ? <MoonIcon /> : <SunIcon />}
      onClick={toggleColorMode}
      p={2}
      data-testid="dark-mode-toggle"
    />
  );
};
