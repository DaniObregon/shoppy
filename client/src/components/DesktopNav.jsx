import React from "react";
import {
  Box,
  Stack,
  Link,
  Popover,
  PopoverTrigger,
  PopoverContent,
  useColorModeValue,
} from "@chakra-ui/react";
import { useSelector } from "react-redux";
import { useNavigate } from "react-router-dom";
import { DesktopSubNav } from "./DesktopSubNav";
import { NAV_ITEMS } from "../utils/navItems";

export const DesktopNav = () => {
  const linkColor = useColorModeValue("gray.600", "gray.200");
  const linkHoverColor = useColorModeValue("gray.800", "white");
  const popoverContentBgColor = useColorModeValue("white", "gray.800");

  // Obtener el rol del usuario desde Redux
  const { role_id } = useSelector((state) => state.userInfo);
  const navigate = useNavigate();

  const handleNavigation = (path) => {
    navigate(path);
  };

  return (
    <Stack direction={"row"} spacing={4}>
      {NAV_ITEMS(role_id).map((navItem) => (
        <Box key={navItem.label}>
          <Popover trigger={"hover"} placement={"bottom-start"}>
            <PopoverTrigger>
              <Link
                p={2}
                onClick={() => handleNavigation(navItem.path)}
                fontSize={"m"}
                fontWeight={500}
                color={linkColor}
                _hover={{
                  textDecoration: "none",
                  color: linkHoverColor,
                }}
              >
                {navItem.label}
              </Link>
            </PopoverTrigger>

            {navItem.children && (
              <PopoverContent
                border={0}
                boxShadow={"xl"}
                bg={popoverContentBgColor}
                p={4}
                rounded={"xl"}
                minW={"sm"}
              >
                <Stack>
                  {navItem.children.map((child) => (
                    <DesktopSubNav key={child.label} {...child} />
                  ))}
                </Stack>
              </PopoverContent>
            )}
          </Popover>
        </Box>
      ))}
    </Stack>
  );
};
