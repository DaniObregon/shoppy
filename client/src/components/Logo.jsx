import React from "react";
import logo from "../assets/butterfly-02.png";
import { useNavigate } from "react-router-dom";
import { Image } from "@chakra-ui/react";

export const Logo = () => {
  const navigate = useNavigate();

  const handleNavigate = () => {
    navigate("/logo-navigation");
  };
  return (
    <Image
      src={logo}
      alt="Logo"
      boxSize="50px"
      marginLeft={3}
      marginRight={-5}
      cursor="pointer"
      onClick={handleNavigate}
      
    />
  );
};
