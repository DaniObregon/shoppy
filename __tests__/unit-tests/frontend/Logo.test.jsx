import "@testing-library/jest-dom";
import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import { ChakraProvider } from "@chakra-ui/react";
import { Logo } from "../../../client/src/components/Logo";
import { MemoryRouter, useNavigate } from "react-router-dom";

// Mock de useNavigate para simular redirección
jest.mock("react-router-dom", () => ({
  ...jest.requireActual("react-router-dom"),
  useNavigate: jest.fn(),
}));

// Mock de la imagen para evitar errores de importación
jest.mock("../../../client/src/assets/butterfly-02.png", () => "logo-mock.png");

describe("Logo component", () => {
  test("Should render Logo component with expected structure", () => {
    render(
      <ChakraProvider>
        <MemoryRouter>
          <Logo />
        </MemoryRouter>
      </ChakraProvider>
    );

    // Verifica que la imagen del logo está en el documento
    const logo = screen.getByRole("img", { name: /logo/i });
    expect(logo).toBeInTheDocument();
  });

  test("Should navigate to '/logo-navigation' when clicked", () => {
    const mockNavigate = jest.fn();
    useNavigate.mockReturnValue(mockNavigate);

    render(
      <ChakraProvider>
        <MemoryRouter>
          <Logo />
        </MemoryRouter>
      </ChakraProvider>
    );

    const logo = screen.getByRole("img", { name: /logo/i });
    fireEvent.click(logo);

    // Verifica que navigate fue llamado con el path correcto
    expect(mockNavigate).toHaveBeenCalledWith("/logo-navigation");
  });

  test("should match snapshot", () => {
    const { asFragment } = render(
      <ChakraProvider>
        <MemoryRouter>
          <Logo />
        </MemoryRouter>
      </ChakraProvider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
