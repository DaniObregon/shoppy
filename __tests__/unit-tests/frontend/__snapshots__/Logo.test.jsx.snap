// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should match snapshot 1`] = `
<DocumentFragment>
  <img
    alt="Logo"
    class="chakra-image css-12p8lrt"
    src="logo-mock.png"
  />
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
